product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/edb_ddl_extractor.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: "Extracting schemas using the EDB DDL Extractor"
+navTitle: Using EDB DDL Extractor
+---
+
+
+Download the latest EDB DDL Extractor script from the Migration Portal Projects page.
+
+!!! Note
+    Migration Portal might become unresponsive for very large SQL files, depending on your system and browser resource availability. To resolve this, try extracting fewer schemas at a time to reduce the SQL file size.
+
+## Prerequisites
+
+You can run the EDB DDL Extractor script in SQL Developer or SQL\*Plus. It uses Oracle’s `DBMS_METADATA` built-in package to extract DDLs for different objects under schemas (specified while running the script). The EDB DDL Extractor creates the DDL file uploaded to the portal and analyzed for EDB Postgres Advanced Server compatibility.
+
+!!! Note
+    You must have `CONNECT` and `SELECT_CATALOG_ROLE` roles and `CREATE TABLE` privilege.
+
+### For SQL\*Plus
+
+1.  Connect to SQL\*Plus and run this command:
+
+    ```sql
+    @edb_ddl_extractor.sql
+    ```
+
+1.  When prompted, if the Oracle user that is running the script has the required privileges, press **Enter** to continue running the script. 
+
+1.  When prompted, provide the schema name. When extracting multiple schemas, use a comma (`‘,’`) as a delimiter.
+
+ !!! Note
+     If you want to extract all the user schemas from the current database, don't mention any schema names while extracting. However, we recommend mentioning the schema names that you want to extract.
+
+1.  When prompted, enter the path or directory for the extractor to store the extracted DDL. If you don't specify a path, the extracted DDL is output to the directory where you ran the SQL\*Plus command.
+  
+1.  When prompted, enter `yes`or `no` depending on whether you want to extract dependent objects from other schemas.
+
+1.  When prompted, enter `yes`or `no` depending on whether you want to extract grant statements from other schemas.
+
+## Worked example
```

Comment:
"Worked example" seems like an awkward title. Was this meant to be "Working example" instead? That would probably be better. However, I think it would be sufficient for the section title to be "Example".